remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingAbstract.java

```diff
@@ -381,7 +380,6 @@ protected RPCHook getRPCHook() {
      * </p>
      */
     public void scanResponseTable() {
-        final List<ResponseFuture> rfList = new LinkedList<ResponseFuture>();
```

Comment:
I am confused because it still removes non-timeout responseFuture

Reply:
My bad,now I have fixed it.Also I have a question about the operation of releasing the responseFuture,is it a redundant operation?

Reply:
In fact,"(rep.getBeginTimestamp() + rep.getTimeoutMillis() + 1000)" means the responseFuture's The latest timeout time,if this time bigger than current time, it means this responseFuture is not timeout.
In this fix,I remove the operation which casue recall the method executeInvokeCallback() for timeout responseFuture.

Reply:
Why do we remove the executeInvokeCallback() method here ? If it timesout doesn't mean it should't execute InvokeCallback. Because timeout is also a form of completion,.

Reply:
> Why do we remove the executeInvokeCallback() method here ? If it timesout doesn't mean it should't execute InvokeCallback. Because timeout is also a form of completion,.

I agree with your option.I will close this PR.